src/components/ScriptureWorkspaceCard.js

```diff
@@ -94,12 +94,17 @@ export default function ScriptureWorkspaceCard({
   }
   
   let title = '';
-  const idParts = id.split('-');
-  if ( BIBLE_AND_OBS[bookId] ) {
-    title += BIBLE_AND_OBS[bookId];
-    idParts.shift()
+  // const idParts = id.split('-');
+  if ( BIBLE_AND_OBS[bookId.toLowerCase()] ) {
+    title += BIBLE_AND_OBS[bookId.toLowerCase()];
+    // idParts.shift()
+  }
+  // title += ' ('+idParts.join('-')+')'
```

Comment:
Can we remove the commented out code
```suggestion
  if ( BIBLE_AND_OBS[bookId.toLowerCase()] ) {
    title += BIBLE_AND_OBS[bookId.toLowerCase()];
  }
```